specs/reject.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { describe, it } from "./_describe.ts"
+import { reject, curry } from '../mod.ts'
+import { eq } from "./utils/utils.ts"
+
+describe('reject', () => {
+  const equals = curry(2, (x: number, y: number) => x === y)
+  let even = (x: number) => x % 2 === 0
```

Comment:
for even we can also use - 
` (x: number) => (x & 1) === 0`
and for odd  - 
` (x: number) => (x & 1) === 1`

current is also fine..just for info..this is more efficient